x/concentrated-liquidity/position_test.go

```diff
@@ -2314,6 +2316,13 @@ func (s *KeeperTestSuite) TestTransferPositions() {
 			outOfRangePositions: []uint64{DefaultPositionId + 1, DefaultPositionId + 2},
 			positionsToTransfer: []uint64{DefaultPositionId, DefaultPositionId + 2},
 		},
+		"three position IDs, not an owner of one of them but caller is gov address": {
```

Comment:
Since this is a gov module acc as the sender, it owns none of them. Is it not?

Reply:
You are right, thanks :)